bin/maintenance/build-wheel.py

```diff
@@ -235,22 +235,35 @@ def cli(obj, target_dir):
 @click.option('--no-assets', 'assets', is_flag=True, flag_value=False, default=True, help='skip building assets')
 @click.option('--add-version-suffix', is_flag=True, help='Add a local suffix (+yyyymmdd.hhmm.commit) to the version')
 @click.option('--ignore-unclean', is_flag=True, help='Ignore unclean working tree')
+@click.option('--no-git', is_flag=True, help='Skip checks and assume clean non-git working tree')
 @click.pass_obj
-def build_indico(obj, assets, add_version_suffix, ignore_unclean):
+def build_indico(obj, assets, add_version_suffix, ignore_unclean, no_git):
     """Build the indico wheel."""
     target_dir = obj['target_dir']
-    # check for unclean git status
-    clean, output = git_is_clean_indico()
-    if not clean and ignore_unclean:
-        warn('working tree is not clean [ignored]')
-    elif not clean:
-        fail('working tree is not clean', verbose_msg=output)
-    # check for git-ignored files included in the package
-    clean, output = package_is_clean_indico()
-    if not clean and ignore_unclean:
-        warn('package contains unexpected files listed in git exclusions [ignored]')
-    elif not clean:
-        fail('package contains unexpected files listed in git exclusions', verbose_msg=output)
+    if no_git:
+        clean, output = True, None
+        warn('Assuming clean non-git working tree')
+        if add_version_suffix: 
+            fail('The --no-git option cannot be used with --add-version-suffix', verbose_msg=output)
```

Comment:
I don't think `verbose_msg=output` makes sense here - `output` is always `None` anyway.

```suggestion
            fail('The --no-git option cannot be used with --add-version-suffix')
```